schemas/com.snowplowanalytics.snowplow.media/player/jsonschema/1-0-0

```diff
@@ -0,0 +1,131 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a context entity for media events that describes the media player and playback state",
+  "self": {
+    "vendor": "com.snowplowanalytics.snowplow.media",
+    "name": "player",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "currentTime": {
+      "type": "number",
+      "description": "The current playback time",
+      "minimum": 0,
+      "maximum": 2147483647
+    },
+    "duration": {
+      "type": [
+        "number",
+        "null"
+      ],
+      "description": "A double-precision floating-point value indicating the duration of the media in seconds",
+      "minimum": 0,
+      "maximum": 2147483647
+    },
+    "ended": {
+      "type": "boolean",
+      "description": "Whether playback of the media has ended"
+    },
+    "fullscreen": {
+      "type": [
+        "boolean",
+        "null"
+      ],
+      "description": "Whether the video element is fullscreen"
+    },
+    "isLive": {
```

Comment:
Maybe drop `is` to be consistent with the other flags? Maybe `livestream` or something?

Reply:
True, have changed to `livestream` to be consistent with other boolean properties.